straxen/contexts.py

```diff
@@ -190,11 +188,12 @@ def xenonnt_online(output_folder='./strax_data',
 
     # Add the rucio frontend if we are able to
     if HAVE_ADMIX:
-        rucio_frontend = straxen.rucio.RucioFrontend(include_remote=True,
-                                                     staging_dir=output_folder,
-                                                     download_heavy=download_heavy
-                                                    )
-        st.storage.insert(0, rucio_frontend)
+        rucio_frontend = straxen.rucio.RucioFrontend(
+            include_remote=True,
+            staging_dir=os.path.join(output_folder, 'rucio'),
+            download_heavy=download_heavy,
+        )
+        st.storage += [rucio_frontend]
```

Comment:
This should solve https://github.com/XENONnT/straxen/pull/688#discussion_r728742966

[Some code relies ](https://github.com/XENONnT/olmo/blob/master/online_monitor/online_monitor.py#L364) (I admit not ideal) on the rundb being the first SF in the context. I think that making sure the two frontends (the datadir with the output path and the rucio frontend) don't mix would make things easier to debug if we get some issue (like #586 and #670)